lambda_web_scraper/climatechange_academic_handler.py

```
@@ -0,0 +1,158 @@
+from datetime import datetime, timedelta
+import pytz
+from typing import Dict, Any
+from common_utils import (
+    fetch_page,
+    get_recent_notices,
+    save_notices_to_db,
+    send_slack_notification,
+)
+
+
+def handler(event, context):
```

Ruff (0.12.2)

Unused function argument: `event`

(ARG001)